cylc/flow/scheduler_cli.py

```diff
@@ -44,35 +44,39 @@
 
 PLAY_DOC = r"""cylc [control] play [OPTIONS] [ARGS]
 
-Start a workflow run, or resume from the previous state if the workflow was
-previously run, or resume a paused workflow.
+Start running a workflow, or restart a stopped workflow from its previous
+state/cycle point, or resume a paused workflow by releasing all tasks.
 
 The scheduler will run as a daemon unless you specify --no-detach.
 
-If the suite is not already installed (by "cylc install" or a previous run)
+If the workflow is not already installed (by "cylc install" or a previous run)
 it will be installed on the fly before start up.
 
 Examples:
-    # Run/resume the workflow with name REG.
+    # Start/restart the workflow with name REG.
     $ cylc play REG
 
-    # Install $PWD/flow.cylc as $(basename $PWD) and run it.
-    # Run/resume the workflow in $PWD.
+    # Install $PWD/flow.cylc as $(basename $PWD) and start it.
     $ cylc play
 
-A "cold start" (the default for a fresh run) starts from the initial cycle
-point (specified in flow.cylc or on the command line). Any dependence on tasks
-prior to the suite initial cycle point is ignored.
+A "cold start" (the default for a freshly-installed workflow) starts from the
+initial cycle point (specified in flow.cylc or on the command line). Any
+dependence on tasks prior to the initial cycle point is ignored.
 
-A "warm start" starts from a given cycle point (--startcp=CYCLE_POINT) that is
-later than the initial cycle point specified in flow.cylc. The
-initial cycle point is preserved, but all tasks and dependencies before the
-given warm start cycle point are ignored.
+A "warm start" starts (a freshly-installed workflow) from a given cycle point
+(--startcp=CYCLE_POINT) that is later than the initial cycle point specified in
+flow.cylc. The initial cycle point is preserved, but all tasks and dependencies
+before the given start cycle point are ignored.
```

Comment:
I'm not sure we need to explain the difference between cold and warm starts in the docstring anymore, maybe just explain the `--startcp` option (or maybe not at all, as that's in the help anyway?)

Reply:
We could stop using the terminology "warm start", but still need to explain that you can start afresh mid-way through the graph.